tests/kuttl/test-suites/default/watcher-decision-engine/05-assert.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+namespaced: true
+commands:
+  - script: |
+      set -ex
+      echo "foo"
```

Comment:
No need to add a fake assert file I think. You could add a 05-errors.yaml to check that the watcherapi CR has been removed.

Reply:
ack doing